package.json

```diff
@@ -15,7 +15,14 @@
   },
   "version": "0.1.0",
   "devDependencies": {
+    "@babel/core": "^7.9.0",
```

Comment:
I didn't change any code that caused these to need to be added, it's happening for me on master too. If I don't add these, my assets don't compile anymore locally, but they're causing deployment to fail. I will need to investigate more, but any insight you have would be right.

Reply:
That's odd. I'm not getting anything like this. Have you tried clearing `node_modules`?

Reply:
Yes, I tried that, unfortunately. When I come back to this I will try reinstalling or upgrading yarn, some people online said that might help. Thanks.